staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer.go

```diff
@@ -226,3 +236,17 @@ func getSvcScheme(svc *v1.Service) cloud.LbScheme {
 	}
 	return cloud.SchemeExternal
 }
+
+func checkMixedProtocol(ports []v1.ServicePort) error {
```

Comment:
there's a MixedProtocolNotSupported constant defined in the API:

```
	// MixedProtocolNotSupported error in PortStatus means that the cloud provider
	// can't ensure the port on the load balancer because mixed values of protocols
	// on the same LoadBalancer type of Service are not supported by the cloud provider.
	MixedProtocolNotSupported = "MixedProtocolNotSupported"
```

is this adding that properly in versions where mixed protocols are not supported, or just skipping without setting loadbalancer status on the service?

Reply:
it seems that is only defined in the internal API , I have to move it to the external API to be consumed by the external cloud providers
```
grep -r MixedProtocolNotSupported pkg/
pkg/apis/core/types.go: // MixedProtocolNotSupported error in PortStatus means that the cloud provider
pkg/apis/core/types.go: MixedProtocolNotSupported = "MixedProtocolNotSupported"
```

Reply:
fixed